### PR TITLE
Adds query architecture, working, build steps and image source in README

### PR DESCRIPTION
**Pull request by JBAhire**
This PR,
- Adds query architecture diagram and explains entity service in it's context
- Adds build steps and image source in README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,4 +3,24 @@
...
+Service that provides CRUD operations for differently identified entities of observed applications.
+
+## How do we use Entity service?
+
+| ![space-1.jpg](https://hypertrace-docs.s3.amazonaws.com/HT-query-architecture.png) | 
+|:--:| 
+| *Hypertrace Query Architecture* |
+
+- Core concepts like relationships between entities, their attributes are defined in this layer but this layer is generic and doesn't really care about what's inside each entity. 
+- Entity service registers the attribute metadata with AttributeService.
```
Comment (buchi-busireddy) [inline README.md:15]: This doesn't already happen.

File: README.md [hunk 2]
```diff
@@ -3,4 +3,23 @@
...
\ No newline at end of file
+Service that provides CRUD operations for differently identified entities of observed applications.
+
+## How do we use Entity service?
+
+| ![space-1.jpg](https://hypertrace-docs.s3.amazonaws.com/HT-query-arch.png) | 
+|:--:| 
+| *Hypertrace Query Architecture* |
+
+- Core concepts like relationships between entities, their attributes are defined in this layer but this layer is generic and doesn't really care about what's inside each entity.
```
Comment (kotharironak) [inline README.md:14]:
```suggestion
- A service layer manages a life cycle of the identified entities of observed applications.
```

File: README.md [hunk 3]
```diff
@@ -3,4 +3,23 @@
...
+Service that provides CRUD operations for differently identified entities of observed applications.
+
+## How do we use Entity service?
+
+| ![space-1.jpg](https://hypertrace-docs.s3.amazonaws.com/HT-query-arch.png) | 
+|:--:| 
+| *Hypertrace Query Architecture* |
+
+- Core concepts like relationships between entities, their attributes are defined in this layer but this layer is generic and doesn't really care about what's inside each entity. 
+- Entity service is also responsible for scalability of entity persistence and CRUD APIs.
```
Comment (kotharironak) [inline README.md:15]:
```suggestion
- Provides CRUD operations for raw or enriched entities, for its types, and their relations
```

**Review by kotharironak — DISMISSED** [review]

File: README.md [hunk 4]
```diff
@@ -3,4 +3,23 @@
...
+Service that provides CRUD operations for differently identified entities of observed applications.
+
+## How do we use Entity service?
+
+| ![space-1.jpg](https://hypertrace-docs.s3.amazonaws.com/HT-query-arch.png) | 
+|:--:| 
+| *Hypertrace Query Architecture* |
+
+- A service layer manages a life cycle of the identified entities of observed applications.
+- Provides CRUD operations for raw or enriched entities, for its types, and their relations
```
Comment (kotharironak) [inline README.md:15]:
```suggestion
- Provides CRUD operations for raw or enriched entities, for its types, and their relations.
```

**Review by kotharironak — APPROVED** [review]

**Pull request merged**